redash/models/__init__.py

```diff
@@ -474,11 +473,11 @@ class Query(ChangeTrackingMixin, TimestampMixin, BelongsToOrgMixin, db.Model):
     last_modified_by = db.relationship(User, backref="modified_queries", foreign_keys=[last_modified_by_id])
     is_archived = Column(db.Boolean, default=False, index=True)
     is_draft = Column(db.Boolean, default=True, index=True)
-    schedule = Column(MutableDict.as_mutable(PseudoJSON), nullable=True)
-    interval = pseudo_json_cast_property(db.Integer, "schedule", "interval", default=0)
+    schedule = Column(MutableDict.as_mutable(JSON), nullable=True)
```

Comment:
Is there a downside to using JSONB here?

Reply:
yes, we can. does it make sense to use JSONB instead of all JSON fields, especially for ones, that are searchable

Reply:
Great; I think this is good to go after converting remaining JSON to JSONB

We can make the change to `query_results.data` as a separate change if you'd like

Reply:
@eradman converted `query_results.data`, all `options` and other json columns to jsonb